### DISCUSSION TITLE
The menu doesn't appear after installation complete

### DISCUSSION THREAD

**Discussion by Setiawan-byte** [BlafKing/sd-civitai-browser-plus#104] [Q&A]
hi
i am using SD automatic 1111
I installed sd-civitai-browser-plus from the web UI - extension, after installation completed and relaunched the web UI Civitai menu didn't appear.
![Screenshot 2023-11-22 152239](https://github.com/BlafKing/sd-civitai-browser-plus/assets/57568443/f1204ade-a3d7-43ab-b0c7-2a295e35f154)

**Comment by BlafKing** [comment]
This is likely caused by you not having all required packages installed in python yet.

You can try restarting the application which should automatically install all requirements to see if it fixes your issue.

If not, please make a bug report with all the requested info! :)

**Reply by Setiawan-byte** [reply]
it appear like this
may i know what packages to installed in python.

Restarting UI...
Closing server running on port: 7860
Tag Autocomplete: Could not locate model-keyword extension, Lora trigger word completion will be limited to those added through the extra networks menu.
[-] ADetailer initialized. version: 23.11.1, num models: 9
CivitAI Browser+: Aria2 RPC started
Error loading script: civitai_gui.py
Traceback (most recent call last):
  File "D:\stableDiffussion\stable-diffusion-webui\modules\scripts.py", line 263, in load_scripts
    script_module = script_loading.load_module(scriptfile.path)
  File "D:\stableDiffussion\stable-diffusion-webui\modules\script_loading.py", line 10, in load_module
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "D:\stableDiffussion\stable-diffusion-webui\extensions\sd-civitai-browser-plus\scripts\civitai_gui.py", line 2, in <module>
    from modules import script_callbacks, shared, ui_settings
ImportError: cannot import name 'ui_settings' from 'modules' (unknown location)

2023-11-22 16:17:32,064 - ControlNet - INFO - ControlNet v1.1.419
Warning: ControlNet failed to load SGM - will use LDM instead.
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Startup time: 3.8s (load scripts: 3.2s, create ui: 0.4s, gradio launch: 0.1s).
HTTP Request: POST http://127.0.0.1:7860/api/predict "HTTP/1.1 200 OK"
HTTP Request: POST http://127.0.0.1:7860/api/predict "HTTP/1.1 200 OK"
HTTP Request: POST http://127.0.0.1:7860/reset "HTTP/1.1 200 OK"
HTTP Request: POST http://127.0.0.1:7860/reset "HTTP/1.1 200 OK"
HTTP Request: POST http://127.0.0.1:7860/api/predict "HTTP/1.1 200 OK"
HTTP Request: POST http://127.0.0.1:7860/reset "HTTP/1.1 200 OK"
![Screenshot 2023-11-22 164137](https://github.com/BlafKing/sd-civitai-browser-plus/assets/57568443/0591958b-1ced-4175-9f5d-44489d5a6408)

**Reply by Setiawan-byte** [reply]
Oh i need to update from the launcher now its good

**Reply by BlafKing** [reply]
What exactly do you mean with

> Oh i need to update from the launcher now its good

Was wondering since someone else is having this same issue :)

**Reply by Setiawan-byte** [reply]
Automatic 1111 needs to be updated because somehow the auto-update (_on my launcher_) didn't work.
sd-civitai-browser-plus need to import "ui_settings" from the modules, but it is only available on A1111 v1.6.0
![Screenshot 2023-11-27 105852](https://github.com/BlafKing/sd-civitai-browser-plus/assets/57568443/4d3830c5-64bf-47ef-a8ab-e2a2baf4c1f4)
